parm/config/config.anal

```diff
@@ -121,12 +121,12 @@ if [[ $RUN_ENVIR == "emc" ]]; then
 
 
 #   NOTE:
-#   As of 2020052612, gfsv16_historical/global_convinfo.txt.2020052612 is
+#   As of 2021110312, gfsv16_historical/global_convinfo.txt.2021091612 is
 #   identical to ../global_convinfo.txt.  Thus, the logic below is not
 #   needed at this time.
 #   Assimilate COSMIC-2 GPS
```

Comment:
This should probably be consistent and say "Assimilate DO-3 Spire" instead of the previous COSMIC-2, even if it's not being used yet.